### PR TITLE
Skipped Warnings

### PR DESCRIPTION
**Pull request by oops00**
Skipped Warnings

### CODE REVIEW

**Comment by oops00** [comment]
in the "validation.h" in the constructor of the Validation class, the skip parameter is specified, which by default leaves warnings in the log file. Change the value to true or specify this parameter when defining the class

File: src/jrd/validation.h [hunk 1]
```diff
@@ -160,7 +162,7 @@ class Validation
 	void checkDPinPIP(jrd_rel *relation, ULONG page_number);
 
 public:
-	explicit Validation(thread_db*, Firebird::UtilSvc* uSvc = NULL);
+	explicit Validation(thread_db*, Firebird::UtilSvc* uSvc = NULL, bool Skipped = false);	// Edited by AIR
```
Comment (oops00) [inline src/jrd/validation.h:165]: explicit Validation(thread_db*, Firebird::UtilSvc* uSvc = NULL, bool Skipped = true);	// Edited by AIR

**Comment by hvlad** [comment]
The main problem of this patch is that there is no way for end user to set (or clear) "skip warns" flag.
The second problem is that code makes no respect to existing code conventions and contains few not needed formatting changes.

**Comment by hvlad** [comment]
Could you, please, don't push every commit here ? 
Push *one* finall common commit when you consider patch finished.
Thanks.

**Pull request closed without merging**